src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -312,6 +318,13 @@ public void execute() throws GitException, InterruptedException {
                 warnIfWindowsTemporaryDirNameHasSpaces();
 
                 launchCommandWithCredentials(args, workspace, cred, url, timeout);
+
+                if (withLFS) {
+                    ArgumentListBuilder lfsArgs = new ArgumentListBuilder();
+                    lfsArgs.add("lfs");
```

Comment:
Wouldn't it be better to check that LFS is available in the git version being used, rather than just throw an exception when it fails?